src/acl.c

```diff
@@ -270,7 +272,7 @@ static int setAclDataCallback(int bufferSize, char *buffer, void *callbackData)
 }
 
 
-static void setAclCompleteCallback(S3Status requestStatus, 
```

Comment:
I really have to figure out why my emacs configuration doesn't tidy up extraneous spaces automatically ...